src/routers/upload.route.ts

```diff
@@ -0,0 +1,12 @@
+import { Router } from 'express';
+import { uploadCustomersController } from '../upload/controllers/uploadCustomers.js';
+import { uploadCarsController } from '../upload/contrllers/uploadCars.js';
+import multer from 'multer';
+
+const router = Router();
+const upload = multer({ dest: 'uploads/' });
+
+router.post('/customers', upload.single('file'), uploadCustomersController);
+router.post('/cars', upload.single('file'), uploadCarsController);
```

Comment:
<img width="303" height="81" alt="image" src="https://github.com/user-attachments/assets/ac091239-a440-4324-af49-f5e7c663845f" />


프론트 api가 `/cars/upload`, `/customers/upload`이라서 수정이 필요할 것 같아요



```suggestion
router.post('/customers/upload', upload.single('file'), uploadCustomersController);
router.post('/cars/upload', upload.single('file'), uploadCarsController);
```